charts/nfs-subdir-external-provisioner/templates/clusterrolebinding.yaml

```diff
@@ -14,3 +14,25 @@ roleRef:
   name: {{ template "nfs-subdir-external-provisioner.fullname" . }}-runner
   apiGroup: rbac.authorization.k8s.io
 {{- end }}
+
+
+{{- if .Values.scc.create }}
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  labels:
+    app: {{ template "nfs-subdir-external-provisioner.name" . }}
+    chart: {{ template "nfs-subdir-external-provisioner.chart" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+  name: scc-hostmount-{{ template "nfs-subdir-external-provisioner.fullname" . }}
```

Comment:
if i already have the current version of the provisioner installed and i already fixed the ClusterRoleBinding myself by hand or by using the commands from the readme: the upgrade to version that contains this addition will be faultless?